### ISSUE TITLE
add keyword to the `plot` function to only show specific branches.

### ISSUE THREAD

**Issue by oameye** [NonlinearOscillations/HarmonicBalance.jl#74]
Would be nice to have a feature to only plot specific branches of the solutions. For example, the following would only plot branch 1 and 3 from the "physical" amplitude plot.
```julia
plot_amplitude = plot(result_amplitude, x="ω", y="sqrt(u1^2 + v1^2)", branches = [1,3])
```
![image](https://user-images.githubusercontent.com/57623933/198384211-f02a3707-3e79-4d00-849a-5773ecc2d237.png)

**Comment by jkosata** [comment]
This is definitely needed, I wrote it for myself once but it got lost somewhere in the commit swamp

Temporarily, one may extract the relevant data from `transform_solutions`

**Comment by oameye** [comment]
Will be implemented after #67.
